serving/ingress/pkg/reconciler/ingress/resources/route.go

```diff
@@ -111,6 +111,26 @@ func makeRoute(ci *networkingv1alpha1.Ingress, host string, rule networkingv1alp
 		return nil, ErrNoValidLoadbalancerDomain
 	}
 
+	terminationPolicy := routev1.InsecureEdgeTerminationPolicyAllow
+	if ci.Spec.HTTPOption == networkingv1alpha1.HTTPOptionRedirected {
+		terminationPolicy = routev1.InsecureEdgeTerminationPolicyRedirect
+	}
+
+	// TODO: Remove this annotation handling after serving 0.26+.
+	// Ingress configures the HTTPOption based on the annotation.
+	// https://github.com/knative/serving/commit/d9c1342b5761afdac88c563535885e37fae27c7e
+	if annotations[networking.HTTPOptionAnnotationKey] != "" {
+		annotation := annotations[networking.HTTPOptionAnnotationKey]
+		switch strings.ToLower(annotation) {
+		case "enabled":
+			terminationPolicy = routev1.InsecureEdgeTerminationPolicyAllow
+		case "redirected":
+			terminationPolicy = routev1.InsecureEdgeTerminationPolicyRedirect
+		default:
```

Comment:
Is `"disabled"` no a valid value here?

Reply:
Yeah, we can support `disable` on Downstream but upstream does not support it and the `disable` annotation is invalid value. So I think we should align with upstream and we can consider to support it if customers requested in the future.

Reply:
ack, okay!